README.md

```diff
@@ -196,65 +206,74 @@ To run all examples and play around with the code you have to run two separate c
 ## Migrating from v0 to v1
 
 ### `init` method signature has changed
+
 - `applicationID` is now called `appId`, to stay consistent with our [other js libraries](https://www.algolia.com/doc/guides/building-search-ui/upgrade-guides/js/?language=javascript#previous-usage).
 
 **Before**:
+
```

Comment:
There are lots of changes right now because I run prettier.
They should be solved when this PR is merged: https://github.com/algolia/search-insights.js/pull/108 and the conflicts solved.

I'll leave this PR as draft until then

Reply:
it's done, prettier pr is merged. README.md diff is smaller now.